docs/dev/cli/cli-commands.md

```diff
@@ -92,6 +92,15 @@ Options explanation:
 ![Class Diagram](img/example_usage.png)
 ***Note:*** Example usage of the CLI for Platform Informations
 
+## Change API Url
```

Comment:
You could use the term endpoint instead of Url

Reply:
What is about the idea to give the user commands to set the endpoint? It is a far better solution then navigate manually to the config folder.